.github/workflows/anonymization-backend-release.yml

```diff
@@ -0,0 +1,125 @@
+name: release
```

Comment:
We already have several release workflows, could you name it something more specific?

Reply:
Sorry! It was copy-pasta from a smaller test 🙂 Fixed now (named `anonymization-backend-release` like the file).